docs/paper/dev/api/inventory-holder.md

```diff
@@ -0,0 +1,156 @@
+---
+slug: /dev/inventory-holder
+---
+
+# InventoryHolder
+
+Using an InventoryHolder is a great way of identifying your plugin's inventories in events.
+
+## Why use an InventoryHolder?
+
+InventoryHolder greatly simplifies the steps you need to do to make sure an inventory was created by your plugin.
+
+Using inventory names for identification is unreliable as other plugins can create inventories with names exactly as yours
+and with Components you need to make sure the name is exactly the same or serialize it to other formats.
+
+InventoryHolder has no such downsides and by using it you're guaranteed to have methods available to handle your inventory.
+
+## Creating an InventoryHolder
+
+InventoryHolder is an interface that we must implement.
+We can do this the following way, let's create a new class that will create our `Inventory` in the constructor.
+
+:::info
+
+The constructor takes your main plugin class as an argument in order to create the `Inventory`,
+if you wish you can use the static method `Bukkit.createInventory(InventoryHolder, int)` instead and remove the argument.
+
+:::
+
+```java
+public class MyInventory implements InventoryHolder {
+
+	private final Inventory inventory;
+
+	public MyInventory(MyPlugin plugin) {
+		// Create an Inventory with 9 slots, `this` here is our InventoryHolder.
+		this.inventory = plugin.getServer().createInventory(this, 9);
+	}
+
+	@Override
+	public Inventory getInventory() {
+		return this.inventory;
+	}
+
+}
+```
+
+## Opening the inventory
+
+To open the inventory, first we have to instantiate our `MyInventory` class and then open the inventory for the player.
+You can do that wherever you need.
+
+:::note
+
+We pass an instance of our plugin's main class as it's required by the constructor. If you've used the static method and removed the constructor
+argument you don't have to pass it here.
+
+:::
+
+```java
+Player player; // Assume we have a Player instance.
+               // This can be a command, another event or anywhere else you have a Player.
+
+MyInventory myInventory = new MyInventory(myPlugin);
+player.openInventory(myInventory.getInventory());
+```
+
+## Listening to an event
+
+Once we have the inventory open, we can listen to any inventory events we like and check if the `Inventory#getHolder()`
+returns an instance of our `MyInventory`.
+
+```java
+@EventHandler
+public void onInventoryClick(InventoryClickEvent event) {
+	Inventory inventory = event.getInventory();
+	// Check if the holder is our MyInventory,
+	// if yes, use instanceof pattern matching to store it in a variable immediately.
+	if(!(inventory.getHolder() instanceof MyInventory myInventory)) {
+		// It's not our inventory, ignore it.
+		return;
+	}
+
+	// Do what we need in the event.
+}
+```
+
+## Storing data on the InventoryHolder
+
+Imagine that we need to store some data for an inventory, if we're using InventoryHolders that
+becomes extremely easy.
```

Comment:
Remove "extremely easy". Rephrase this to sound less opinionated (I think that's the word to describe this). Something like `You can store extra information on an Inventory if you use an InventoryHolder.`